src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costRent = 40;
+  const discountThreeDays = 20;
+  const discountForDaysThree = 3;
+  const discountSevenDays = 50;
+  const discountForDaysSeven = 7;
```

Comment:
```suggestion
  const rentCost = 40;
  const smallDiscount = 20;
  const daysForSmallDiscount = 3;
  const bigDiscount = 50;
  const daysForBigDiscount = 7;
```

do not put number values to the var naming